trpcultivate_phenotypes/src/Plugin/TripalImporter/TripalCultivatePhenotypesTraitsImporter.php

```diff
@@ -503,8 +505,10 @@ public function formValidate($form, &$form_state) {
         // ********************************************************************
         // Skip empty lines
         else if (!empty(trim($line))) {
-          // Split line into an array
-          $data_row = str_getcsv($line, "\t");
+          // Split line into an array using the delimiter defined by this importer
+          // in the configure values method above.
+          $data_row = TripalCultivatePhenotypesValidatorBase::splitRowIntoColumns($line, $file_mime_type);
```

Comment:
Yup, good catch. I added the call to this where it will eventually be used to make sure we remember: https://github.com/TripalCultivate/TripalCultivate-Phenotypes/pull/101/files#diff-67c6ba893a67e6b0852eebeafa9c18ea19db7d7c7fa68a22530f746f526e89b1R485